lec15-data-viz.Rmd

```diff
@@ -1,4 +1,341 @@
+
 ---
-title: "Data visualization in R"
+title: Data visualization in R
+author: Lindsay Coome
 ---
 
+## Lesson preamble
+
+> ### Learning Objectives
+>
+> - Making better plots for outreach and communication.
+> - Review how to produce scatter plots with trendlines using ggplot.
+> - Advanced faceting and segmenting data.
+> - Altering asthetic features in ggplot.
+> - Make bar and other graphs using ggplot.
+> - Making interactive and quick graphs.
+> - Saving graphs.
+> 
+> ### Lesson outline
+>
+> - How to visualize data (20 min)
+> - ggplot basics (10 min)
+> - Basics and faceting (10 min)
+> - Aesthetics and plot types (25 min)
+> - Interactive and quick graphing (10 min)
+> - Saving graphs (5 min)
+
+-----
+
+## What makes a good graph?
+> #### Challenge
+>
+> What makes a bad graph? What makes an ugly graph to you?
+>
+
+
+There are many considerations to take into account when creating a graph.
+Data types:
+* Nominal
+* Ordinal
+* Interval
+* Ratio
+And data features:
+* Scalar
+* Vector
+* Tensor
+
+Ask: "what kind of data/information is it?"
+* Numerical?
+* What type?
+* Spatial?
+* How many dimensions?
+* Nominal/categorical?
+* Narrative?
+
+## Tufte's guidelines
+* Reduce non-data ink
```

Comment:
Maybe a short explanation of who Tufte is

Reply:
will be explained verbally

Reply:
Yes, but it would be nice for the website to have a link to him and his work.

Reply:
Yes, it's good to have as much as possible explained in the lecture content.

Reply:
Added resource